### PR TITLE
[Blog] Meet the Maintainer: Michael Gfeller

### PR DESCRIPTION
**Pull request by leecalcote**
Signed-off-by: leecalcote <leecalcote@gmail.com>

**Description**

1st in a blog series...

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
@mgfeller how are we looking here?

**Comment by mgfeller** [comment]
@leecalcote we're looking real good here :smile:  just one detail as mentioned in a comment I didn't submit (but did right now).

File: src/collections/blog/2021-02-04-meet-the-maintainer-michael-gfeller/index.mdx [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+</p></div>
+
+<div class="interviewer">
+<span>Shriti:</span><p>
+Michael, thank you for joining me today. Many people inside and outside of the Layer5 Community have seen the affects of your contributions, but may not know the backstory as to who Michael is and how you arrived at your maintainer role. Indulge us. How did you come across the Layer5 community? What made you stay?
+</p></div>
+
+<div class="interviewee">
+<span>Michael:</span><p>
+"Hey Shriti, thank you for having me. If I remember correctly, I was looking for a professional focus area earlier this year, and landed on service meshes. I had worked with service meshes before, which helped piqued peak my initial interest. I had worked with Istio as part of my work for some time, and in the same context I stumbled across Meshery. I wasn’t able to dig deep in Meshery back then, but the concept sounded fascinating to me. Additionally, I had been benefiting from open source, both personally and professionally for a long time, but apart from a few minor contributions, I had never been able to give back to the community. I hoped to correct that."
```
[the remark this answers is not in the document]
Comment (leecalcote) [reply src/collections/blog/2021-02-04-meet-the-maintainer-michael-gfeller/index.mdx:37]: Yes, indeed. Spot on. Updating...

**Comment by leecalcote** [comment]
Perfect. Thank you, Michael. What a great article. 💯

**Comment by mgfeller** [comment]
> Perfect. Thank you, Michael. What a great article. 100

Teamwork with @chandrashritii !

**Pull request merged**